testdata/example-config-gc.json

```diff
@@ -128,5 +128,6 @@
     "archive": false,
     "gc_blocks_limit": 42,
     "gc_fork_clean_step": 420,
+    "num_epochs_to_keep_store_data": 5,
```

Comment:
Could you set it to 24 so we test that it’s set properly in the test?

Reply:
Done.